### ISSUE TITLE
setup assumes an init.d based sysmanager instead of systemd?

### ISSUE THREAD

**Issue by eriknelson** [hbons/Dazzle#47]
OS: Fedora 32 for Raspberry Pi:

This one is a little odd because most major distributions at this point are systemd based and should use `systemctl sshd reload`.

```
[root@<host> sparkleshare]# DAZZLE_USER=sparkle DAZZLE_GROUP=sparkle DAZZLE_HOME=/nfsmnt/sync ./dazzle.sh setup
 1/4 | Installing the Git package...
  -> The Git package has already been installed (version 2.26.2).
 2/4 | Creating account "sparkle"...
  -> Account already exists.
  -> mkdir --parents /nfsmnt/sync
mkdir: cannot create directory ‘/nfsmnt’: File exists
 3/4 | Configuring account "sparkle"...
  -> mkdir --parents /nfsmnt/sync/.ssh
mkdir: cannot create directory ‘/nfsmnt’: File exists
  -> touch /nfsmnt/sync/.ssh/authorized_keys
touch: cannot touch '/nfsmnt/sync/.ssh/authorized_keys': No such file or directory
  -> chmod 700 /nfsmnt/sync/.ssh
chmod: cannot access '/nfsmnt/sync/.ssh': No such file or directory
  -> chmod 600 /nfsmnt/sync/.ssh/authorized_keys
chmod: cannot access '/nfsmnt/sync/.ssh/authorized_keys': No such file or directory
 4/4 | Reloading the SSH config...
  -> /etc/init.d/ssh reload
./dazzle.sh: line 116: /etc/init.d/ssh: No such file or directory

Setup complete!
To create a new project, run "dazzle create PROJECT_NAME".
```

**Comment by eriknelson** [comment]
Same issue #37

**Comment by eriknelson** [comment]
Fixed by #48
